docs/hotkeys.md

```diff
@@ -83,10 +83,20 @@
 |Autocomplete @mentions, #stream_names, :emoji: and topics|<kbd>Ctrl</kbd> + <kbd>f</kbd>|
 |Cycle through autocomplete suggestions in reverse|<kbd>Ctrl</kbd> + <kbd>r</kbd>|
 |Narrow to compose box message recipient|<kbd>Meta</kbd> + <kbd>.</kbd>|
+
```

Comment:
In this commit, it'd be helpful to clarify where the keys in the new categories come *from*. The code suggests that (via the change in category tags), but it's helpful if the commit text (or even title) mentions that.

Note that if a commit is merged to `main`, you can reference it in the commit text here, ie. where all the editors became readline-capable.